bin/setup_dev

```diff
@@ -147,25 +112,26 @@ if [ -n "$SIMPLE" ]; then
     exit 0
 fi
 
+rm -rf protoc-gen-doc
+git clone https://github.com/pseudomuto/protoc-gen-doc.git
+
 bin/setup_remote faucet
 bin/setup_remote forch
 bin/setup_remote udmi
 
-echo -n "DAQ commit "
-git log -n 1 --pretty=format:"%h - %an, %ar : %s" || true
-echo
+bin/setup_mininet
```

Comment:
i think mininet is also needed for the "simple" setup mode

Reply:
moved